crypto/ec/ec_asn1.c

```diff
@@ -24,7 +24,7 @@ int EC_GROUP_get_basis_type(const EC_GROUP *group)
 
     /* Find the last non-zero element of group->poly[] */
     for (i = 0;
-         i < (int)OSSL_NELEM(group->poly) & group->poly[i] != 0;
+         i < (int)OSSL_NELEM(group->poly) && group->poly[i] != 0;
```

Comment:
> it will go away on rebase since @levitte aleady fixed it.

Still, unrelated changes to separate commits. Now this request would *have to* be re-based, normally one would have to wait out ci...